redirects-transition.conf

```diff
@@ -1120,6 +1127,9 @@ rewrite ^/eeo/(.*).pdf https://www.fec.gov/about/equal-employment-opportunity/ r
 rewrite ^/elecfil/authorized_manual/(.*) https://efilingapps.fec.gov/fecfiledoc/fecfiledoc.pdf redirect;
 rewrite ^/elecfil/formats/(.*) https://efilingapps.fec.gov/registration/softwarelogs.htm redirect;
 
+# em/enfpro broader redirects
+rewrite ^/em/enfpro/Enforcement_Matters_(.*) https://www.fec.gov/legal-resources/enforcement/enforcement-profile/ redirect;
```

Comment:
let's make this more specific to PDF files just in case. So it should look like

`rewrite ^/em/enfpro/Enforcement_Matters_(.*).pdf https://www.fec.gov/legal-resources/enforcement/enforcement-profile/ redirect;`